test/libsolidity/Metadata.cpp

```diff
@@ -400,6 +400,55 @@ BOOST_AUTO_TEST_CASE(metadata_revert_strings)
 	BOOST_CHECK_EQUAL(metadata["settings"]["debug"]["revertStrings"], "strip");
 }
 
+BOOST_AUTO_TEST_CASE(metadata_optimiser_sequence)
+{
+	char const* sourceCode = R"(
+		pragma solidity >=0.0;
+		contract test {
```

Comment:
I'd call it `C` or `A` because then it stands out in calls below as a contract name :) `test` is a bit generic in this context.
```suggestion
		contract C {
```

Reply:
We have a few more `contract test` instances here as well. Will make a mental note to clean it up in another PR, together with the `std::string` stuff. I have changed this particular instance to `contract C` though.

Reply:
Yeah. I agree. Better to leave cleaning up the old stuff for later. I try to point this stuff out only when I see it in new code anyway because that's always an opportunity to do things better :)